ddtrace/contrib/fastapi/__init__.py

```diff
@@ -0,0 +1,45 @@
+"""
+The FastAPI__ integration is provided by a ASGI middleware.
+
+To configure tracing manually::
+
+    from ddtrace.contrib.asgi import TraceMiddleware
+
+    app = FastAPI()
+    app.add_middleware(TraceMiddleware)
+
+If using Python 3.6, the legacy ``AsyncioContextProvider`` will have to be
+enabled before using the middleware::
+
+    from ddtrace.contrib.asyncio.provider import AsyncioContextProvider
+    from ddtrace import tracer  # Or whichever tracer instance you plan to use
+    tracer.configure(context_provider=AsyncioContextProvider())
```

Comment:
if the ordering matters, should we show a full example here? (e.g. where does `app = FastAPI()` go?)